Employment Policies/Working Remotely.md

```diff
@@ -1,11 +1,91 @@
+## Approach
+
+Fundamentally, working remotely is a privilege that comes with both benefits and costs. We should treat it as such.
+
+The benefits are primarily ones of flexibility: as individuals, we are able to work when we want to and where we want to. This lets us do our best work and contribute to our team in a custom-crafted framework that lets us lead more fulfilling lives. The costs are primarily ones of communication: being out of the office means more base-line work and incidental complexity for all employees. More is required of both the remote employee and the employees in the office to keep in sync, collaborate, and own the work we do in a visible way. 
+
+When we work remotely, it's our responsibility to accept the benefits graciously, and go above and beyond to not let the costs affect our work or the work of our teammates. A remote workday:
+
+* is not shorter than a day at the office
+* is not less productive
+* is not an excuse for missing commitments (be they meetings or deadlines). 
+
+Even when we aren't in the office, our teammates are still counting on us and it is our responsibility to make sure we are exceeding their expectations.
+
+At the same time, it is the responsibility of workers in the office to reasonably accommodate remote workers. This means being willing to schedule meetings in light of timezone differences and to manage expectations around immediate feedback.  
+
+The guidelines in this document (and the handbook at large) give us tools to make remote work easier, but tooling is only a small part of the solution. All employees need to complement these tools with the extra work required to make being out of the office an effective thing for both the individual remote worker and the team of people who rely on them.
+
+## Policies
+
+The majority of policies that support remote work in Clef apply to everyone at Clef, not just remote workers. The primary documents that outline these policies are:
+
+* [Effective Meetings](https://github.com/clef/handbook/blob/master/Operations%20Documents/Effective%20Meetings.md)
+* [Communication and Transparency](https://github.com/clef/handbook/blob/master/Onboarding%20Documents/Communication%20and%20Transparency.md)
+
+### Extended Remote Work
+
+#### Extended remote work 
+
+***An extended remote work period includes anything longer than 2 days or any period of time where we work from a irregular place.***
+
+If you're planning to work remotely for more than two consecutive days, or you're planning to work remotely from a place that's abnormal (i.e. from a partner's home in a different city), you'll need to follow these additional guidelines. Even more than standard remote work, working remotely for an extended period of time is a privilege — it **will** require a large amount of extra work from you to make sure that the team is effective with you working remotely.
```

Comment:
What is a place that's abnormal? I started this comment looking for wording clarity, but now realize that I actually don't know what that means.